source/runtime_gui.cpp

```diff
@@ -2528,6 +2538,12 @@ void reshade::runtime::draw_variable_editor()
 }
 void reshade::runtime::draw_technique_editor()
 {
+	bool reload_required = false;
+
+	if (_effect_load_skipping_ui)
+		if (size_t skipped_effects = std::count_if(_effects.begin(), _effects.end(), [](const effect &effect) { return effect.skipped; }); skipped_effects > 0)
+			reload_required = ImGui::ButtonEx(std::format("Force load all effects (%lu remaining)", skipped_effects).c_str(), ImVec2(ImGui::GetWindowContentRegionWidth(), 0));
```

Comment:
Could the `format` call here be replaced with an inline call to either `sprintf_s` or `ImFormatString`? I would like to avoid adding a new header (std_string_ext.hpp).